package.json

```diff
@@ -47,24 +54,28 @@
     "stylint": "1.5.9"
   },
   "dependencies": {
-    "@atlaskit/editor-core": "134.0.1",
+    "@atlaskit/editor-core": "acezard/editor-core-fork#134.0.1",
```

Comment:
Should be hosted by cozy.

Reply:
see https://github.com/cozy/cozy-notes/pull/196#discussion_r682382474